Alignment/OfflineValidation/bin/PVmerge.cc

```diff
@@ -60,21 +60,7 @@ int merge(int argc, char* argv[]) {
   int w_dzPhiNormMax = validation.get_child_optional("w_dzPhiNormMax") ? validation.get<int>("w_dzPhiNormMax") : 1.8;
```

Comment:
The use of `get_child_optional` is a bit strange. Its output is a `boost::optional<int>` object, not a boolean. I know this isn't part of this PR, but we could correct this. The easiest would be to change `get_child_optional` into `count`, which should do the job.

Reply:
Changed it for all validations